### ISSUE TITLE
Wrong fee for partially-fillable orders in the API

### ISSUE THREAD

**Issue by fhenneke** [cowprotocol/services#1500]
Our orderbook api seems to only show the fee of the latest execution of a partially-fillable order in the `executedSurplusFee` field. I would have expected to see the aggregate of all fees.

For example, [this order](https://explorer.cow.fi/orders/0xee838bbf128c79d9dc68a4bc9aa7f395b4a7196ab0dd52881c3054907e32199f00e286b5256aa6cf252d5a8a5a7b8c20ec3bc4d564824da5/?tab=fills) was partially executed twice. The [orderbook](https://api.cow.fi/mainnet/api/v1/orders/0xee838bbf128c79d9dc68a4bc9aa7f395b4a7196ab0dd52881c3054907e32199f00e286b5256aa6cf252d5a8a5a7b8c20ec3bc4d564824da5) shows a `executedSurplusFee` of `5727493546878075`.
According to our data base,
```
select auction_id, surplus_fee from order_execution where order_uid = '\xee838bbf128c79d9dc68a4bc9aa7f395b4a7196ab0dd52881c3054907e32199f00e286b5256aa6cf252d5a8a5a7b8c20ec3bc4d564824da5';

 auction_id |    surplus_fee
------------+-------------------
    6317763 | 10983658878257777
    6321462 |  5727493546878075

```
this is only the latest fees.

**Comment by nlordell** [comment]
It seems that we don't have logic for summing the total surplus fees from all order executions (i.e. the current DB query assumes fill-or-kill semantics for the fee computation). This will be tricky...

**Comment by fhenneke** [comment]
This seems to have been fixed by #1866.
